src/protocolsupportpocketstuff/hacks/itemframes/ItemFramesPacketListener.java

```diff
@@ -1,27 +1,25 @@
-package protocolsupportpocketstuff.hacks.itemframes;
+/*package protocolsupportpocketstuff.hacks.itemframes;
```

Comment:
Seems a bit too hacky, I think, just going with the axe like that. Or do you want to pull this in first, and then clean up the itemframes handling?